src/test/regress/sql/pg14.sql

```diff
@@ -39,6 +39,15 @@ reindex(verbose, TABLESPACE test_tablespace) index idx ;
 reindex(TABLESPACE test_tablespace1) index idx;
 set citus.log_remote_commands to off;
 
+-- ALTER STATISTICS .. OWNER TO CURRENT_ROLE errors out
```

Comment:
this test shows our approach has something missing. How come the same command works on PG but not on Citus? Why can't we get the `CURRENT_ROLE` and PG can get it?